src/components/collect-button.tsx

```diff
@@ -51,15 +51,9 @@ export class CollectButtonComponent extends BaseComponent<IProps, IState> {
     );
   }
   private handleClickButton = () => {
-   this.addTestMouseToBackpack();
-  }
-  private addTestMouseToBackpack = () => {
```

Comment:
Is it worth leaving the addtestMouseToBackpack function around as a potential tool for future use?  I'm ok either way, but it could be useful to have a function that adds a test mouse to the backpack.

Reply:
Sure, we can leave it. I'm not sure that the button view will be the best place to leave the orphaned function, but we we could leave it regardless.